### PR TITLE
Style form

### PR DESCRIPTION
**Pull request by mariawright05**
# Describe your changes 

Added styles:
- button hover
- input text style
- checkbox cursor
- file upload cursors
- Submit button goes to redirect

I also threw in the gitignore/package-lock changes

# I follow best practices 

- [ ] My code is covered by unit tests 
- [x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [x] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [x] I have tested my PR locally and proven it works. 
- [x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

File: frontend/src/components/startup-form/styledStartupForm.js [hunk 1]
```diff
@@ -99,7 +106,13 @@ export const Button = styled.button`
   line-height: 30px;
   font-weight: bold;
   border: none;
+  transition: transform ease-in 0.2s;
   box-shadow: inset -3px 3px 4px #AB69FF, inset 4px -4px 4px rgba(56, 0, 128, 0.4), 0 24px 18px -12px rgba(74, 0, 169, 0.4);
+  &:hover {
```
Comment (rileydanejohnston) [inline frontend/src/components/startup-form/styledStartupForm.js:111]: Awesome!

**Comment by rileydanejohnston** [comment]
Looks good! Take a look at the comments regarding the picture inputs. It may be something you're saving for a future design I want to make sure before I approve.

File: frontend/src/components/startup-form/StartupForm.js [hunk 2]
```diff
@@ -15,21 +16,27 @@ import {
...
         <StartupTitle>Igor Shoifot is in search of startups</StartupTitle>
         <LargeCaption>Fill in this form and the investor will see your startup in his feed</LargeCaption>
       </HeadingContainer>
-      <Form>
+      <Form onSubmit={onSubmit}>
         <LogoLabel>
           Startup Logo*
           <LogoInput
-            type="text"
+            type="file"
```
[the remark this answers is not in the document]
Comment (mariawright05) [reply frontend/src/components/startup-form/StartupForm.js:39]: Yes, styling file inputs is a little more complicated than I thought! I am working on this, but won't be ready until after the demo. I should have indicated in the notes!

**Review by rileydanejohnston — APPROVED** [review]
Looks great!

**Pull request merged**